docs/src/App.js

```diff
@@ -5,7 +5,7 @@ import Content from './Content'
 
 import 'typeface-roboto/index.css'
 
-import 'css/reset.css'
+import '@dhis2/ui-core/build/es/css/reset.css'
```

Comment:
This is not great. Gonna solve this in a different way.

Reply:
See my PR comment for another approach to this